docs/source/faq/index.rst

```diff
@@ -80,24 +97,225 @@ of these popular operating systems from the past few years.
 We also provide :ref:`custom install instructions <alternative-builds>` to use
 FiftyOne on old-but-popular setups like Ubuntu 16.04 and Debian 9.
 
-Can you share a dataset with someone else?
-------------------------------------------
+.. _faq-share-dataset-export:
+
+Can I share a dataset with someone else?
+----------------------------------------
 
-You can easily :doc:`export a dataset </user_guide/export_datasets>` in one
```

Comment:
Let's prefer to use `.. _anchor-text:` definitions in the docs and link to them via `:ref:` rather than linking via `:doc: </path/to/file>`, since the former will work even if we decide to reorganize content in the docs, rename pages, change the canonical source for a certain topic, etc.